docs/apache-airflow/installation/index.rst

```diff
@@ -0,0 +1,335 @@
+if you look for longer discussion and have more information to share.
+
+If you can provide description of a reproducible problem with Airflow software, you can open
+issue at `GitHub issues <https://github.com/apache/airflow/issues>`__
+
+
+Using Production Docker Images
+''''''''''''''''''''''''''''''
+
+More details: :doc:`docker-stack:index`
+
+**When this option works best**
+
+This installation method is useful when you are familiar with Container/Docker stack. It provides a capability of
+running Airflow components in isolation from other software running on the same physical or virtual machines with easy
+maintenance of dependencies.
+
+The images are build by Apache Airflow release managers and they use officially released packages from PyPI and official constraint files
+- same that are used for installing airflow from PyPI.
+
+**Intended users**
+
+Users who are familiar with Containers and Docker stack and understand how to build their own container images.
+
+Users who understand how to install providers and dependencies from PyPI with constraints if they want to extend or customize the image.
+
+Users who now how to create deployments using Docker by linking together multiple docker containers and maintaining such deployments.
+
+**What are you expected to handle**
+
+You are expected to be able to customize or extend Container/Docker images if you want to
+add extra dependencies. You are expected to put together a deployment built of several containers
+(for example using docker-compose) and to make sure that they are linked together.
+
+You are responsible for setting up database, creating and managing database schema with ``airflow db`` commands,
+automated startup and recovery, maintenance, cleanup and upgrades of Airflow and the Airflow Providers.
+
+With the Official Airflow Docker Images, upgrades of Airflow and Airflow Providers which
+are part of the reference image are handled by the community - you need to make sure to pick up
+those changes when released by upgrading the base image. However you are responsible in creating a
+pipeline of building your own custom images with your own added dependencies and Providers and need to
+repeat the customization step and building your own image when new version of Airflow image is released.
+
+You might also want to upgrade your image independently with new Providers when they are released
+independently from releasing the Airflow image.
+
+**What Apache Airflow Community provides for that method**
+
+You have instructions: :doc:`docker-stack:build` on how to build and customize your image.
+
+You also have `quick start guide </start/index>`_ where you can see an example of Quick Start docker-compose which
+you can use to start Airflow quickly for local testing and development. However this is just an inspiration.
+Do not expect this docker-compose is ready for production installation, you need to get familiar
```

Comment:
Here I am not sure when we are talking about the `docker-compose.yaml` file and when we are talking about the Docker compose product.  Google has made recommendations on how to reference files. See: https://developers.google.com/style/filenames

Reply:
I rephrased it.